tasks/main.yml

```diff
@@ -85,6 +93,13 @@
       or kernel_settings_transparent_hugepages_defrag
       or kernel_settings_bootloader_cmdline | d({})
       or kernel_settings_purge
+      or __kernel_settings_cur_header != __kernel_settings_new_header
+  vars:
+    __kernel_settings_cur_header: "{{
+      __kernel_settings_profile_contents_b64.content | b64decode |
+      regex_replace('(?sm)^\\[.*$', '') }}"
```

Comment:
Why are there two backslashes in `'(?sm)^\\[.*$'`?

Reply:
That's apparently how you have to escape the `[` so it won't be interpreted as a character set in this context

Reply:
must be to escape the second backslash, Ansible won't read it otherwise